src/github.com/getlantern/lantern/feed.go

```diff
@@ -160,35 +163,17 @@ func GetFeed(locale string, proxyAddr string, provider FeedProvider) {
 		return
 	}
 
-	processFeed(provider)
+	processFeed(allStr, provider)
 }
 
 // processFeed is used after a feed has been downloaded
 // to extract feed sources and items for processing.
-func processFeed(provider FeedProvider) {
+func processFeed(allStr string, provider FeedProvider) {
```

Comment:
Oh, I see - the feed names are effectively already "translated" because they are what they are.  Thanks.